tools/reccmp/reccmp.py

```diff
@@ -7,6 +7,7 @@
 import subprocess
 import os
 import sys
+import colorama
```

Comment:
You need to install colorama.
Perhaps add a `tools/reccmp/requirements.txt` file, and install this on ci as well.

Reply:
What do you think about not importing colorama if --no-color is passed?

That way CI doesn't have to worry about installing colorama when it doesn't need color to begin with.

Reply:
Personally, I prefer to test things. This avoids silently breaking code.
ci does `pip install capstone` already.

Reply:
Added requirements.txt and updated build.yaml